### DISCUSSION TITLE
Where can I find multi-cluster flat networks support documentation?

### DISCUSSION THREAD

**Discussion by shadiramadan** [linkerd/linkerd2#11271] [General] [answered]
Hello!

I'd be happy to test out https://github.com/linkerd/linkerd2/releases/tag/edge-23.8.3 as I have a flat multi-cluster network setup w/ Private GKE clusters.

But I want to make sure I'm setting things up correctly.  I tried to look through the edge docs but I didn't find anything regarding the changes.
https://linkerd.io/2-edge/overview/

See: https://linkerd.io/2023/07/20/enterprise-multi-cluster-at-scale-supporting-flat-networks-in-linkerd/ for some background.

@mateiidavid @wmorgan @adleong 

https://github.com/linkerd/linkerd2/pull/11224
https://github.com/linkerd/linkerd2/pull/11226

For changes do I need to:
- Add the `multicluster.linkerd.io/remote-discovery` label to all services that I would like to support with cross cluster communication.
- When linking clusters add `--gateway=false` to the command

Is this all that needs to be done?

**Comment by wmorgan** [comment, marked as answer]
@shadiramadan I believe that's right. The docs are coming in hot and as of this moment are mostly in PR form. See e.g. https://github.com/linkerd/website/pull/1663 for the walkthrough. Let us know how it goes!

**Reply by shadiramadan** [reply]
Sweet! Will report back in a few days!
